staging/src/k8s.io/client-go/tools/clientcmd/api/helpers.go

```diff
@@ -27,8 +27,8 @@ import (
 )
 
 func init() {
-	sDec, _ := base64.StdEncoding.DecodeString("REDACTED+")
-	redactedBytes = []byte(string(sDec))
+	sDec, _ := base64.StdEncoding.DecodeString("DATA+OMITTED")
```

Comment:
I think we actually want to keep both of these. The `ClientKeyData` should definitely remain as `REDACTED+`, since it's actually a secret.

Reply:
I'd argue that other secrets like `AuthInfos[*].Token` and `AuthInfos[*].Password` should be redacted too. What do you think?

Reply:
I would agree, but that's not my call to make 😄

Reply:
Okay, I've thought about it, and I think we should scope this PR to _just_ change the public CA to be `DATA+OMITTED`. We can open another PR about the other fields.

It's not my call (I'm not on a sig/* group), but I think that's the best way to avoid contention.

Reply:
Sure.